### ISSUE TITLE
3D render of Earth

### ISSUE THREAD

**Issue by g4ze** [g4ze/byoc#2]
make the Spline render less repeated and constant using layouts in nextjs so that it doesn't load on every new page.

**Comment by anonthedev** [comment]
I suppose this is already done but why? the earth is basically affecting the readibility of the website and increasing the loading time anyways, is it really that necessary?

**Comment by g4ze** [comment]
nope not necessary! do you have any suggestions wrt to the UI of the site? We can discuss on that :)

**Comment by anonthedev** [comment]
something like vercel would be nice

**Comment by g4ze** [comment]
i was tryna go with that only, but left it half done. 
a ui/ux like this is what we aim: https://neon.tech/

**Comment by anonthedev** [comment]
yeah, building a frontend like that would be nice. Though it won't be a piece of cake
